build_projects/dotnet-cli-build/GenerateNuGetPackagesArchiveVersion.cs

```diff
@@ -48,7 +48,7 @@ public override bool Execute()
                 {
                     ToolPath = ToolPath,
                     TemplateType = newArgs[0],
-                    TemplateArgs = newArgs[1] + $" --debug:ephemeral-hive -n TempProject -o \"{outputDir}\"",
+                    TemplateArgs = newArgs[1] + $" --no-restore --debug:ephemeral-hive -n TempProject -o \"{outputDir}\"",
```

Comment:
??  What's the story here?

Reply:
The new template engine is in the 1.1.0 build, which means that dotnet new will automatically run restore for certain projects. However, for this one, it is put under a temp folder and as such does not use our nuget.config with our special feeds for these packages. We don't really need it in this step, as all we do is gather all the packages and use them to create a hash that serves as an ID for the lzma.